examples/todo/js/components/TodoList.js

```diff
@@ -44,23 +44,36 @@ class TodoList extends React.Component {
 }
 
 export default Relay.createContainer(TodoList, {
+  initialVariables: {
+    status: null,
+  },
+
+  prepareVariables({status}) {
+    let nextStatus;
+    if (status === 'active' || status === 'completed') {
+      nextStatus = status;
+    } else {
+      // This matches the Backbone example, which displays all todos on an
+      // invalid route.
+      nextStatus = 'any';
+    }
+    return {status: nextStatus};
+  },
+
   fragments: {
-    todos: () => Relay.QL`
-      fragment on TodoConnection {
+    viewer: () => Relay.QL`
+      fragment on User {
         completedCount,
-        edges {
-          node {
-            complete,
-            id,
-            ${Todo.getFragment('todo')},
+        todos(status: $status, first: 9007199254740991) {
```

Comment:
This isn't very clear. Can we use `Number.MAX_SAFE_INTEGER` instead?

Or even:

```
const MAX_SAFE_NUMBER = Number.MAX_SAFE_NUMBER || 9007199254740991;
```

(If you are worried about IE and Safari, where it might not be defined.)